### ISSUE TITLE
Using "ng generate ..." with a contextual menu on a folder in an Angular project fails

### ISSUE THREAD

**Issue by Maryannah** [nrwl/nx-console#1168]
![image](https://user-images.githubusercontent.com/46892394/139397547-6e136730-31a2-4f97-bf97-0e45e19154ef.png)

- Create an Angular project
- Right click on the "app" folder
- Choose "ng generate ..."
- Select an @angular/component

![image](https://user-images.githubusercontent.com/46892394/139397706-5cfc6fce-5ebb-4afe-8dd0-0ed84b9c2ff2.png)

![image](https://user-images.githubusercontent.com/46892394/139397849-32a0afbd-fdbf-45ae-956f-a18cff190f37.png)

**Comment by balajibadly** [comment]
same error here

**Comment by Cammisuli** [comment]
This is weird. I was trying to fix this today, and I can reproduce this anymore 🤔 I did manage to reproduce it last week, and the only thing that changed between then and now is the release of VsCode 1.62.

Can everyone check again with the latest?

**Comment by Maryannah** [comment]
![image](https://user-images.githubusercontent.com/46892394/140609229-71bb3d8b-efa5-4505-a99d-a8d4e3f48e89.png)

Still the same issue, I also tried on another computer running w11, still happening

**Comment by Cammisuli** [comment]
Hmmm... Ok let me try again on Windows.

**Comment by Maryannah** [comment]
Oh yes, I did not mention, this is an issue on windows, my bad, sorry for wasting your time if you were on Linux !
